src/hb-subset.h

```diff
@@ -203,6 +203,20 @@ hb_subset_input_set_axis_range (hb_subset_input_t  *input,
 				float               axis_max_value,
 				float               axis_def_value);
 
+HB_EXTERN hb_bool_t
+hb_subset_axis_range_from_string (const char *str, int len,
+				  float *axis_min_value,
+				  float *axis_max_value,
+				  float *axis_def_value);
+
+#ifndef HB_NO_SETLOCALE
```

Comment:
You can't use these macros in header files. What's the problem exactly, do you know?

Reply:
oh yeah, I'll remove this wrapper